templates/items/details/details-spell.hbs

```diff
@@ -43,21 +43,20 @@
     </div>
```

Comment:
Some adjustments will need to be made to the spell header because "Class Spellcasting" is long enough to wrap, leading to the header elements to overlap the tabs:

<img width="510" height="190" alt="Screenshot 2025-07-15 at 10 23 18" src="https://github.com/user-attachments/assets/bbd8d3ce-ef42-46ae-8548-4632e97b4252" />